packages/notebook-extension/schema/tracker.json

```diff
@@ -388,6 +388,12 @@
       "description": "Defines the observed bottom margin for the virtual notebook, set a positive number of pixels to render cells below the visible view",
       "type": "string",
       "default": "1000px"
+    },
+    "maxTopBottomCellOutput": {
+      "title": "The maximum number of output cells to display in the top and bottom for cells",
+      "description": "Defines the maximum number of output cells to display in the top and bottom for cells with many outputs. Output in between will be trimmed and not displayed.",
```

Comment:
Maybe add that the output is available by clinking on the message in the middle or similar?

Reply:
i think the message is there (next line)